spec/fig_offchain_prot.tex

```diff
@@ -73,31 +73,49 @@
 								$\hatmT \gets \hatmT \cup \{\tx\}$ \;
 								% issue snapshot if we are leader
 								\If{$\hats = \bar{\mc S}.s \land \hpLdr(\bar{\mc S}.s + 1) = i$}{
-									\Multi{} $(\hpRS,\hatv,\bar{\mc S}.s+1, \hatmT, \tx_{\omega})$ \;
+									\red{$\psi \gets \bot$} \;
+                  					\Multi{} $(\hpRS,\red{\hatv,}\bar{\mc S}.s+1,\hatmT,\red{\psi})$ \;
```

Comment:
I'm not really sure what $\psi$ is, but this seems like it always requests a "Normal" snapshot?

This is problematic.

If there was something to de/commit before, this would ask for "clearing" it ($\eta_\theta \gets \bot$ in `reqSn` handling).

We must not clear anything pending unless we saw the corresponding `incrementTx` or `decrementTx`!